app/src/main/java/com/dankideacentral/dic/MainActivity.java

```diff
@@ -72,9 +80,20 @@ public void failure(TwitterException exception) {
                 Log.d("TwitterKit", "Login with Twitter failure", exception);
             }
         });
-        SupportMapFragment mapFragment = (SupportMapFragment) getSupportFragmentManager()
-                .findFragmentById(R.id.map);
-        mapFragment.getMapAsync(this);
+        toggleButton = (Button) findViewById(R.id.toggle);
+        toggleButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                currentFragment = currentFragment instanceof ItemFragment ?
```

Comment:
In the past I've done it this way... http://stackoverflow.com/questions/9294603/get-currently-displayed-fragment (top answer -- using tags) That way the FragmentManger manages which fragment is being displayed.